frontend/package.json

```diff
@@ -22,6 +22,7 @@
     "react-dom": "17.0.2",
     "react-error-boundary": "3.1.3",
     "react-helmet": "6.1.0",
+    "react-phone-input-2": "^2.15.1",
```

Comment:
Should we need to add external plugin for phone number? Can this be done without external dependency?

Reply:
I think it is better to use a verified library which checks every validation for phone number along with there country codes.

Reply:
@manassahoo-dev 
I have already raised PR - https://github.com/idurar/erp-crm/pull/567

I have just updated the regex. 

I think no need to add external library. 

This will add extra load in app.